src/components/FloatingActionButton/FabPlusIcon.js

```diff
@@ -12,23 +13,45 @@ const propTypes = {
 };
 
 function FabPlusIcon({isActive}) {
-    const theme = useTheme();
-    const animatedValue = useSharedValue(isActive ? 1 : 0);
+    const {textLight, textDark} = useTheme();
+    const sharedValue = useSharedValue(isActive ? 1 : 0);
 
     useEffect(() => {
-        animatedValue.value = withTiming(isActive ? 1 : 0, {
+        sharedValue.value = withTiming(isActive ? 1 : 0, {
             duration: 340,
             easing: Easing.inOut(Easing.ease),
         });
-    }, [isActive, animatedValue]);
+    }, [isActive, sharedValue]);
 
-    const animatedProps = useAnimatedProps(() => {
-        const fill = interpolateColor(animatedValue.value, [0, 1], [theme.textLight, theme.textDark]);
+    // Adapting fill and stroke properties from react-native-svg to be able to animate them with Reanimated
+    const adapter = createAnimatedPropAdapter(
```

Comment:
We could move `adapter` outside of the component so it doesn't recreate on each render.